Source/SwiftLintFramework/Extensions/String+SwiftLint.swift

```diff
@@ -60,4 +57,10 @@ extension String {
         }
         return nil
     }
+
+    internal func nsrangeToIndexRange(nsrange: NSRange) -> Range<Index> {
```

Comment:
I see this method being used by the regex matchers. Since `NSString` is based around UTF-16 code units and not grapheme clusters, wouldn't this give wrong results for certain kinds of strings?